src/consensus/pbft/libbyz/Pre_prepare.cpp

```diff
@@ -34,6 +34,8 @@ Pre_prepare::Pre_prepare(
   rep().view = v;
```

Comment:
Why is this change necessary? Ideally, the consensus should be agnostic to the type of transaction being executed?

Reply:
I'm also curious.

Reply:
spoke off-line and updated the comment